cpp/include/raft/linalg/detail/norm.cuh

```diff
@@ -138,6 +140,15 @@ void colNormCaller(OutType* dots,
   };
 }
 
+template <typename T, bool DevicePointerMode = false>
+void nrm2(
+  raft::resources const& handle, int n, const T* x, int incx, T* result, cudaStream_t stream)
+{
+  cublasHandle_t cublas_h = resource::get_cublas_handle(handle);
+  detail::cublas_device_pointer_mode<DevicePointerMode> pmode(cublas_h);
+  detail::cublasnrm2(cublas_h, n, x, incx, result, stream);
```

Comment:
How does this differ from the other norm functions? Can we just invoke this underneath the existing norm functions? Is it doing anything significantly different? Even better- could we just use the existing norm functions instead of having to call this at all?

Reply:
Hi @cjnolet, I have refactoring the code to use the existing `raft::linalg::norm` functions and have removed the cublasnrm2 wrapper.